src/Widgets/WindowSwitcher/WindowSwitcher.vala

```diff
@@ -64,114 +61,112 @@ public class Gala.WindowSwitcher : CanvasActor {
         unowned var gtk_settings = Gtk.Settings.get_default ();
         granite_settings = Granite.Settings.get_default ();
 
-        unowned var display = wm.get_display ();
-        scaling_factor = display.get_monitor_scale (display.get_current_monitor ());
+        container = new Clutter.Actor () {
+            reactive = true,
+            layout_manager = new Clutter.FlowLayout (Clutter.FlowOrientation.HORIZONTAL)
+        };
 
-        opacity = 0;
+        caption = new Clutter.Text () {
+            font_name = CAPTION_FONT_NAME,
+            ellipsize = END,
+            line_alignment = CENTER
+        };
+        caption.set_pivot_point (0.5f, 0.5f);
```

Comment:
While we're here can you remove pivot point here. We don't scale the caption, so no reason to set it manually.

Reply:
Will do, I just wasn't quite sure what it was used for so I left it there :)